### DISCUSSION TITLE
Imagick darkens all uploaded images - PHP 8.1

### DISCUSSION THREAD

**Discussion by rosmo01** [ImageMagick/ImageMagick#7200] [General]
Hi,

After updating to PHP 8.1, all images uploaded (to Wordpress) are darkened by about 1 F-stop, which makes all images useless to publish. How to change this?

Morten

**Comment by snibgo** [comment]
This is an ImageMagick forum. IMagick is a separate product, an interface between ImageMagick and PHP. It is supported by different people.

**Reply by rosmo01** [reply]
Dyslectic whoopsie there!
